src/popup/router/pages/AccountDetailsTokens.vue

```diff
@@ -0,0 +1,46 @@
+<template>
+  <div
+    class="token-list-wrapper"
+  >
+    <TokensList
+      :search-term="searchTerm"
+    />
+  </div>
+</template>
+
+<script>
+import TokensList from '../components/FungibleTokens/TokensList.vue';
+
+export default {
+  components: {
+    TokensList,
+  },
+  props: {
+    searchTerm: { type: String, default: '' },
+  },
+};
+</script>
+
+<style lang="scss" scoped>
+@use '../../../styles/variables';
+@use '../../../styles/mixins';
+@use '../../../styles/typography';
+
+.token-list-wrapper {
+  position: relative;
+  padding-top: 4px;
+
+  // ::v-deep .tokens-list-item {
```

Comment:
if not used - delete, if nesesary to stay  - then write comment why.

Reply:
Sure, removed.